reactioncommerce/reaction
# Fixes #2863: added delay and loader

Resolves #2863  
Impact: major
Type: bugfix

## Issue
The issue was basically a design issue in which the user wasn't given feedback of the updation of rates when another shipping method was selected.

## Solution
Added a loader which would be shown before the rates are calculated.
But in my testing, whenever I chose another method the calculation was fast enough that the loader wasn't visible in that time.
So I added a delay in the calculation of 200ms.

## Testing
1. Enable Shippo
2. But the `Example product` and go to the checkout screen.
1. Complete steps till filling of address
3. Click on various shipping methods and see the loader over the Review portion.
